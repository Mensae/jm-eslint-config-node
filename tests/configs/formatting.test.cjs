/* eslint-env jest */

const formattingConfig = require('../../configs/formatting.cjs');

describe('formatting', () => {
  it('should match the snapshot', () => {
    expect(formattingConfig).toMatchSnapshot();
  });
});
