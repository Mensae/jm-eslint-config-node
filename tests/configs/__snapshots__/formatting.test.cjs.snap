// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatting should match the snapshot 1`] = `
{
  "env": {
    "es2022": true,
    "node": true,
  },
  "parserOptions": {
    "ecmaVersion": "latest",
    "sourceType": "module",
  },
  "plugins": [
    "n",
  ],
  "reportUnusedDisableDirectives": true,
  "rules": {
    "n/callback-return": "error",
    "n/exports-style": "error",
    "n/file-extension-in-import": "error",
    "n/global-require": "error",
    "n/no-mixed-requires": "error",
    "n/no-process-env": "error",
    "n/no-restricted-import": "error",
    "n/no-restricted-require": "error",
    "n/no-sync": "off",
    "n/prefer-global/buffer": "error",
    "n/prefer-global/console": "error",
    "n/prefer-global/process": "error",
    "n/prefer-global/text-decoder": "error",
    "n/prefer-global/text-encoder": "error",
    "n/prefer-global/url": "error",
    "n/prefer-global/url-search-params": "error",
    "n/prefer-promises/dns": "error",
    "n/prefer-promises/fs": "error",
  },
}
`;
